integrationtests/wildfly-modules/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://jboss.org/schema/arquillian
+        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <defaultProtocol type="Servlet 3.0" />
+
+    <group qualifier="Grid" default="true">
+        <container qualifier="container.active-1" default="true">
+            <configuration>
+                <property name="jbossHome">${basedir}/target/node1</property>
+                <property name="javaHome">${server.jvm}</property>
+                <property name="javaVmArguments">
+                    ${server.jvm.args}
+                    -Djboss.socket.binding.port-offset=100 -Djava.net.preferIPv4Stack=true
+                    -Djgroups.bind_addr=127.0.0.1 -Djgroups.join_timeout=2000
+                </property>
+                <!-- To debug the Arquillian managed application server: -->
+                <!--<property name="javaVmArguments">-Xrunjdwp:transport=dt_socket,address=8000,server=n,suspend=y -Dorg.jboss.remoting-jmx.timeout=300-->
+                    <!-- -Djboss.socket.binding.port-offset=100 -Xmx512m -Djava.net.preferIPv4Stack=true-->
+                    <!-- -Djgroups.bind_addr=127.0.0.1 -Djgroups.join_timeout=2000-->
+                <!--</property>-->
+                <property name="managementPort">10090</property>
+                <property name="serverConfig">standalone-test.xml</property>
+            </configuration>
+        </container>
+        <container qualifier="container.active-2">
```

Comment:
This container is never explicitly used. Previously it was used by the `org.infinispan.test.integration.as.query.*` tests, should we bring those back?

Reply:
I removed those tests because of the changes @gustavonalle and @anistor have made to query.

Reply:
I'll leave the definition there for future additions. It can stay unused